src/components/Typography/Title/styles/Title.style.ts

```diff
@@ -59,7 +60,7 @@ export const StyledTitle4 = styled.h4`
 `;
 
 export const StyledTitle5 = styled.h5`
-    ${titleStyle};
+    ${getDefaultTitleStyles()}
 
     font-size: 16px;
     line-height: 1.5;
```

Comment:
Co tak ještě vytvořit funkci na nastavení velikosti písma, apod. na Title komponentu? Co myslíš? Zdá se ti to jako dobrý nápad? :)